cfme/cloud/instance/__init__.py

```diff
@@ -403,7 +403,7 @@ def power_control_from_cfme(self, *args, **kwargs):
             view = navigate_to(self, 'AllForProvider')
             view.toolbar.view_selector.select('List View')
             try:
-                row = view.paginator.find_row_on_pages(view.entities.table, name=self.name)
+                row = view.paginator.find_row_on_pages(view.entities.elements, name=self.name)
```

Comment:
btw, I don't thik you need all this code, since Entities are already implemented